tests/integration/kubernetes/k8s-attach-handlers.bats

```diff
@@ -25,8 +25,8 @@ setup() {
 	policy_settings_dir="$(create_tmp_policy_settings_dir "${pod_config_dir}")"
 	
 	display_message="cat /usr/share/message"
-	exec_command="sh -c ${display_message}"
-	add_exec_to_policy_settings "${policy_settings_dir}" "${exec_command}"
+	exec_command=(sh -c "${display_message}")
```

Comment:
Just curious, why are we wrapping `sh` with `()` instead of `""` now? Is there a difference functionally?

Reply:
If using:

```
command="/bin/ls"
exec_command="sh -c ${command}"
kubectl exec -- "${exec_command}"
```

The exec would fail because "${exec_command}" is a single argument to kubectl exec, and "sh -c /bin/ls" is not a valid binary to be executed.

However, when using an array of arguments:

```
command="/bin/ls"
exec_command=(sh -c "${command}")
kubectl exec -- "${exec_command[@]}"
```

kubectl exec gets 3 separate command line arguments and the exec works.